util/godfig/src/backend/mod.rs

```diff
@@ -0,0 +1,57 @@
+pub(crate) trait BackendInternalOperations {
```

Comment:
This looks out of place; an accidental commit from the Godfig work?

Reply:
From `Godfig`